site/_docs/deployment-methods.md

```diff
@@ -107,8 +107,7 @@ If you want to maintain Jekyll inside your existing Rails app, [Jekyll-Admin](ht
 
 ## Amazon S3
 
-If you want to host your site in Amazon S3, you can do so with
-[s3_website](https://github.com/laurilehmijoki/s3_website) application. It will
+If you want to host your site in Amazon S3, you can do so by using the [s3_website](https://github.com/laurilehmijoki/s3_website) application. It will
```

Comment:
Looking at the entire file again, it should be fine. However, this brings to mind a question. Does the Jekyll site have a policy on how links should be handled? IMHO, inline links are gross, and if you're doing that, it isn't that much of a stretch to just use HTML for everything, thus negating any advantage you get from using Markdown.

Seems easier to read if things are separated (e.g, reference links). Thoughts?